install/helm/gloo/templates/44-rbac.yaml

```diff
@@ -22,9 +22,14 @@ rules:
   - services
   - pods
   - nodes
-  - secrets
   - namespaces
   verbs: ["get", "list", "watch"]
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  {{/* This is needed as the gateway deployer would need to create / patch the mtls certs if enabled */}}
```

Comment:
is this still needed, or do we just use the certs generated by the mtls certgen job?

Reply:
We use the certs generated by the mtls certgen job, but we need the permissions to create/patch the secrets managed by the deployer in the namespace to which the gloo-proxy is being deployed.